plexus-compilers/plexus-compiler-aspectj/src/main/java/org/codehaus/plexus/compiler/ajc/AspectJCompiler.java

```diff
@@ -565,75 +572,164 @@ private void checkForAspectJRT( List<String> cp )
     private void setSourceVersion( AjBuildConfig buildConfig, String sourceVersion )
         throws CompilerException
     {
-        if ( "11".equals( sourceVersion ) )
-        {
-            buildConfig.getOptions().sourceLevel = ClassFileConstants.JDK11;
-        }
-        else if ( "10".equals( sourceVersion ) )
-        {
-            buildConfig.getOptions().sourceLevel = ClassFileConstants.JDK10;
-        }
-        else if ( "9".equals( sourceVersion ) )
-        {
-            buildConfig.getOptions().sourceLevel = ClassFileConstants.JDK9;
-        }
-        else if ( "1.9".equals( sourceVersion ) )
-        {
-            buildConfig.getOptions().sourceLevel = ClassFileConstants.JDK9;
-        }
-        else if ( "1.8".equals( sourceVersion ) )
-        {
-            buildConfig.getOptions().sourceLevel = ClassFileConstants.JDK1_8;
-        }
-        else if ( "1.7".equals( sourceVersion ) )
+        buildConfig.getOptions().sourceLevel = versionStringToMajorMinor( sourceVersion );
+    }
+
+    /**
+     * Set the target version in ajc compiler
```

Comment:
nit: period and AJC

Reply:
1. I merely copied the Javadoc from above, so this is not my choice of spelling.
2. All of ajc, Ajc and AJC are in use. Actually, AJC is rather rare among us AspectJ developers. Sometimes we use ajc, because that is the name of the executable. Sometimes we use Ajc when comparing it to Javac as a proper name in prose. Actually, AJC is rather synthetic, in order to look more similar to ECJ when comparing those.

There is no right or wrong here. I copied it, I leave it at that. Otherwise I have to change existing comments in the whole module too.

Reply:
I am simply going to call in "AspectJ compiler", because "AJC compiler" actually means "AspectJ compiler compiler" and is redundant. It is just used in two comments, the one I copied and this one. I am not going to add the period, though, because it is a single sentence, basically like a subject line. The existing javadocs are so bad, I am not going to start revising all of them. For 15 years nobody even so much as looked at this code, and now the code review is about comments in POMs and Javadocs.